.github/workflows/full-ci.yml

```diff
@@ -203,6 +203,17 @@ jobs:
       - name: Integration tests
         run: mage integrationfull
 
+      # Etcd can be still running after integration tests when:
+      # 1. pytest recieve SIGALRM (can be caused by pytest-timeout plugin)
+      # 2. when pytest crashes
+      # In both cases finalizers are not invoked, so etcd is not stopped.
+      - name: Kill etcd, if it was left after integration tests
+        if: always()
+        run: |
+          ETCD_TT_BIN=${ETCD_PATH}etcd;
+          ETCD_TT_PID=$( ps aux | grep ${ETCD_TT_BIN} | grep -vw grep | awk '{ print $2 }' );
```

Comment:
Isn't there something like `pgrep` or `pidof` command?

Reply:
I tested those commands locally on macOs, they doesn't seem to find/kill process by path only by binary name.
I don't want to have a possibility that our tests will kill someone else's 'etcd'.

Reply:
`pgrep --full` isn't supported?
`pidof` also works with full paths on Linux:
```
$ pidof /opt/tarantool/tarantool
5418
```

> I don't want to have a possibility that our tests will kill 'etcd' that was not started by our job.

Sure.

Reply:
```
f.terekhin@f-terekhin sharded-queue-enterprise % pgrep --full
pgrep: illegal option -- -
```
pidof is not pre-installed, needs to be installed separately.

Reply:
`pidof` is not pre-installed on macOs. However, `pkill -f` should work in our case. Changed